ceph-ansible.spec.in

```diff
@@ -44,6 +44,13 @@ pushd %{buildroot}%{_datarootdir}/ceph-ansible
   rm group_vars/common-coreoss.yml.sample
   # These untested playbooks are too unstable for users.
   rm -r infrastructure-playbooks/untested-by-ci
+  %if 0%{?fedora} || 0%{?centos}
```

Comment:
Is that a reverse condition?

I'm reading this as 'if not fedora or centos, then' did I get it right?

Reply:
Yes, the problem is that the `%{rhel}` macro is defined on CentOS as well so it is "compatible", so we have to handle it like this :(

Reply:
So centos and fedora won't have all.yml, I think this is an issue for community users.
Can't we lookup /etc/redhat-release instead?

Reply:
On CentOS and Fedora we delete `rhcs.yml.sample`. They will still have `all.yml.sample.`

Reply:
Right sorry  for the confusion

Reply:
No problem! I dislike using the negative conditionals there, it is confusing.

Reply:
I just thought about this. Can we mv rhcs.yml.sample to all.yml.sample also for RHEL?

Reply:
That is another option. Is that more or less confusing to users?